ui/src/services/segmenter/SegmenterStatus.js

```diff
@@ -8,7 +8,7 @@ export const getSegmenterStatus = (segmenter) => {
     },
     active: {
       label: "Active",
-      color: "#017D73",
+      color: "#00BFB3",
```

Comment:
Same comment as `ui/src/services/experiment/ExperimentStatus.js` - is there a way to get rid of the hardcoded color hex?

Reply:
Same as https://github.com/caraml-dev/xp/pull/33#discussion_r969101354 - I've removed the `healthColor` values and made all components consume the new `color` values (that are of the predetermined `primary`/`success`/... colours).